package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "kontena-lens",
-  "productName": "Lens",
-  "description": "Lens - The Kubernetes IDE",
+  "name": "open-lens",
+  "productName": "OpenLens",
+  "description": "OpenLens - The Kubernetes IDE",
```

Comment:
Please change it to "OpenLens - The Open Source IDE for Kubernetes"

Reply:
Changed.